source/documentation/deploying_apps/deploying_docker_image.md

```diff
@@ -0,0 +1,28 @@
+## Deploy a docker image (experimental)
+
+This section explains how to deploy an app from a docker image. The role of the docker image is to serve as a packaging format and the requirements for the app and its runtime environment are the same as for apps deployed using buildpacks. Configuration via manifest.yml also stays the same, albeit you should not specify buildpack. If you do, it will be ignored, which can create some confusion to anyone reading the manifest, but not knowing app is being deployed from a docker image. Our platform currently only supports deploying images from docker hub or [Docker Trusted Registry](https://docs.docker.com/datacenter/dtr/2.1/guides/) [external page] servers. You cannot deploy from local images.
+
+To deploy app using docker image stored in docker hub via CF cli:
+
+``cf push myapp --docker-image dockerhuborg/app_image:tag``
+
+To deploy app using docker image stored in Docker Trusted Registry (DTR) via CF cli:
+
+``cf push myapp --docker-image MY-PRIVATE-REGISTRY.DOMAIN:5000/image/name:tag``
+
+You might notice that staging process is simplified when deploying from docker image - this is because there is no need to build the app from the pushed code using a buildpack. Also, buildpack for apps deployed from docker image will be reported as `unknown` in app information (``cf app <myapp>``).
+
+There are few more specifics about docker image support in Cloud Foundry:
+
+* Local storage in each container instance is ephemeral. Every container launch will start with unmodified image.
+* Traffic will be routed to the lowest exposed port (by `EXPOSE` command in Dockerfile). PORT environment variable will be also set to this number. 
+* Buildpack based apps use bash to execute the app start command (`bash -c <command>`), but docker image based apps use sh (`sh -c <command>`) - as bash presence cannot be assumed in your docker image.
+* Privileged container mode is not supported. Features depending on this will not work.
+* Only registries that use Docker Registry API V2 are supported.
+* There is currently no support for deploying images from private/non DTR registries.
+
+### Apps from docker images in production
```

Comment:
We should call this something else, because the current title doesn't reflect the importance and doesn't only apply to production apps. Maybe "Differences in responsibility model" or "Applying security updates"?